pkg/server/server.go

```diff
@@ -157,9 +158,17 @@ func (host *yamlLanguageHost) Run(ctx context.Context, req *pulumirpc.RunRequest
 			return err
 		}
 		defer loader.Close()
-
+		conf := make(map[string]string)
+		// Strip the project prefix
+		projPrefix := pctx.Project() + ":"
+		for k, v := range req.GetConfig() {
+			if strings.HasPrefix(k, projPrefix) {
+				k = strings.Split(k, projPrefix)[1]
+			}
```

Comment:
There is a built in function that does this:
```suggestion
			k = strings.TrimPrefix(k, projPrefix)
```